models/model.games.js

```diff
@@ -38,6 +38,25 @@ exports.fetchUsers = () => {
   });
 };
 
+exports.fetchCommentsByReviewId = (review_id) => {
+  return Promise.all([db.query(`SELECT * FROM comments WHERE review_id = $1;`, [review_id]), this.checkReviewIdExists(review_id)]).then(([{ rows }, reviewExists]) => {
+    if (rows.length !== 0 || (rows.length === 0 && reviewExists)) {
+      return rows;
+    } else return Promise.reject({ status: 404, msg: 'Comment not found' });
```

Comment:
You can probably remove this logic as your `checkReviewExists` function will reject when the review does not exist. This means your if condition will always be true and you can just return the rows in all cases.